client/css/self-check.css

```diff
@@ -209,75 +203,82 @@ textarea {
 
 /* The Modal (background) */
 .modal {
-    display: none; /* Hidden by default */
-    position: fixed; /* Stay in place */
-    z-index: 1; /* Sit on top */
-    left: 0;
-    top: 0;
-    width: 100%; /* Full width */
-    height: 100%; /* Full height */
-    overflow: auto; /* Enable scroll if needed */
-    background-color: #909090; /* Fallback color */
-    background-color: rgba(90, 90, 90, 0.9); /* Black w/ opacity */
+	/* display: none; */
+	/* Hidden by default */
+	position: fixed;
+	/* Stay in place */
+	z-index: 1;
+	/* Sit on top */
+	left: 0;
+	top: 0;
+	width: 100%;
+	/* Full width */
+	height: 100%;
+	/* Full height */
+	overflow: auto;
+	/* Enable scroll if needed */
+	background-color: #909090;
+	/* Fallback color */
+	background-color: rgba(90, 90, 90, 0.9);
+	/* Black w/ opacity */
 }
 
 /* Modal Content/Box */
 .modal-content {
-    background-color: #fefefe;
-    margin: 15% auto; /* 15% from the top and centered */
-    padding: 20px;
-    border: 1px solid #888;
+	background-color: #fefefe;
+	margin: 15% auto;
+	/* 15% from the top and centered */
+	padding: 20px;
+	border: 1px solid #888;
 	width: 30%;
-    border-radius: 10px;
+	border-radius: 10px;
 }
 
 .modalheaderdiv {
-    text-align: center;
-    font-weight: bold;
-    font-size: 120%;
+	text-align: center;
+	font-weight: bold;
+	font-size: 120%;
 }
 
-.modalclose {
+.modalclosemain {
 	padding: 10px 40px;
 
 	color: #ffffff;
 	font-weight: bold;
 
 	border-radius: 5px;
 	border: 1px solid #909090;
-	
+
 	background: #656565;
 	background: -webkit-linear-gradient(#808080, #505050);
 	background: -o-linear-gradient(#808080, #505050);
 	background: -moz-linear-gradient(#808080, #505050);
 	background: linear-gradient(#808080, #505050);
-	
-	cursor: pointer; 
+
+	cursor: pointer;
 }
 
-.modalclose:hover {
+.modalclosemain:hover {
 
 	background: #404040;
 	background: -webkit-linear-gradient(#505050, #202020);
 	background: -o-linear-gradient(#505050, #202020);
 	background: -moz-linear-gradient(#505050, #202020);
 	background: linear-gradient(#505050, #202020);
-	
-}
 
-/* The Close Button */
-.close {
-    color: #aaa;
-    float: right;
-    font-size: 16px;
-    font-weight: bold;
 }
 
-.close:hover,
-.close:focus {
-    color: black;
-    text-decoration: none;
-    cursor: pointer;
+/* The Close Button */
+.modalclosex {
+	color: #aaa;
+	float: right;
+	font-size: 16px;
+	font-weight: bold;
 }
 
-
+.modalclosex:hover,
+.modalclosex:focus {

```

Comment:
Same as line 272 - the selector is a class selector with state, but the html element only has the ID.

Reply:
Addressed by 4d00a20d